app/javascript/packages/address-search/components/in-person-locations.tsx

```diff
@@ -18,20 +20,19 @@ interface InPersonLocationsProps {
   locations: FormattedLocation[] | null | undefined;
   onSelect;
   address: string;
+  resultsHeaderComponent?: ComponentType;
 }
 
-function InPersonLocations({ locations, onSelect, address }: InPersonLocationsProps) {
+function InPersonLocations({
+  locations,
+  onSelect,
+  address,
+  resultsHeaderComponent: HeaderComponent,
+}: InPersonLocationsProps) {
   const isPilot = locations?.some((l) => l.isPilot);
 
   if (locations?.length === 0) {
```

Comment:
also late to this, but but I think this will wrong if `locations` is ever null/undefined? because `undefined === 0` is false

Reply:
Similarly expecting to be non-blocking. Also, not technically changing in this release.

While your reasoning is correct, I think it might be intentional? i.e. if `locations` is `undefined`, it might mean that they're being loaded, and we wouldn't want to show a "no results found" message?

cc @allthesignals re: #7468

Reply:
The current usages of the `InPersonLocations` React component (in this release) check whether `locations` is truthy before attempting to render it:
- [in-person-location-full-address-entry-post-office-search-step.tsx](https://github.com/18F/identity-idp/blob/stages/rc-2023-09-12/app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.tsx#L123)
- [address-search.tsx](https://github.com/18F/identity-idp/blob/stages/rc-2023-09-12/app/javascript/packages/address-search/components/address-search.tsx#L46)

Therefore this issue doesn't have practical implications for this release.